examples/firebase/index.js

```diff
@@ -0,0 +1,179 @@
+const functions = require("firebase-functions");
+const { PKPass } = require("passkit-generator");
+const admin = require("firebase-admin");
+var fs = require('file-system');
+var path = require('path');
+var axios = require('axios');
+var os = require('os');
+
+// Firebase init
+admin.initializeApp({
+    credential: admin.credential.cert(require("CERTIFICATE_PATH")),
```

Comment:
What is `CERTIFICATE_PATH` and what is `STORAGE_BUCKET_URL`? Should they be set from `process.env` perhaps?

Reply:
They're both to do with the firebase admin tools to access cloud storage – see https://firebase.google.com/docs/storage/admin/start
You get them both from the Firebase console, I had just pasted in the values and so when pushing this I simply replaced them with placeholders, sorry 😅
I'm not entirely sure what `process.env` is or how it works, but if that's the proper way it should be done please let me know and I'd be happy to try implementing :)

Reply:
I've to check better how this works. I'll get back to you soon.

Reply:
I've removed `CERTIFICATE_PATH` and `STORAGE_BUCKET_URL` for a single `setupData.json` file